modules/virt-building-real-time-container-disk-image.adoc

```diff
@@ -3,163 +3,92 @@
 // * virt//support/monitoring/virt-running-cluster-checkups.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="virt-building-real-time-container-disk-image_{context}"]
-= Building a container disk image for {op-system-base} virtual machines
+[id="virt-building-realtime-container-disk-image_{context}"]
+= Building a realtime container disk image for {op-system-base} virtual machines
```

Comment:
The term `real-time` is all over the documentation.
Please consider leaving it as-is.